wremnants/HDF5Writer.py

```diff
@@ -372,11 +374,12 @@ def write(self,
                     logger.debug(f"Now at proc {proc}!")
 
                     hvar = dg.groups[proc].hists["syst"]
+                    #hnom = dg.groups[proc].hists[chanInfo.nominalName] ## this doesn't work here for some reason, perhaps I should reload the datagroups
```

Comment:
This works once you remove the lines above as mentioned by the previous comment

Reply:
Right, it does, I am changing it